spec/models/defence_request_spec.rb

```diff
@@ -17,6 +17,29 @@
         subject.appropriate_adult = true
       end
       it { expect(subject).to validate_presence_of :appropriate_adult_reason }
+
```

Comment:
Why didn't you use `validate_inclusion_of` Shoulda helper? It should cover all the 3 contexts bellow.